src/components/CandidateAvatar/CandidateAvatar.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from '../../packages/bem';
+import styles from './CandidateAvatar.scss';
+
+const { block, elem } = bem({
+    name: 'CandidateAvatar',
+    classnames: styles,
+    propsToMods: ['context']
+});
+
+const context = percentage => {
+    if (percentage <= 33) {
+        return 'bad';
+    }
+    if (percentage <= 66) {
+        return 'warning';
+    }
+    return 'good';
+};
+
+const CandidateAvatar = props => {
+    const { imageUrl, matchPercentage, showPercentageOnHover, size, ...rest } = props;
+
+    const fixedSize = Math.round(Math.max(32, size));
+    const radius = fixedSize / 2 - 2;
+    const percentage = Math.max(0, Math.min(100, matchPercentage));
+    const circumference = 2 * radius * Math.PI;
+    const strokeDasharray = `${(percentage * circumference) / 100} 999`;
+
+    return (
+        <div
+            {...rest}
+            {...block(props)}
+            style={{
+                width: fixedSize,
+                height: fixedSize
+            }}
+        >
+            <div
+                {...elem('image', props)}
+                style={{
+                    backgroundImage: `url(${imageUrl})`
+                }}
+            >
+                {!!showPercentageOnHover && (
+                    <div {...elem('percentage', props)}>{`${percentage}%`}</div>
+                )}
+            </div>
+            <svg
+                {...elem('ring', props)}
+                style={{
+                    width: fixedSize,
+                    height: fixedSize
+                }}
+            >
+                <circle
+                    r={radius}
+                    cx={fixedSize / 2}
+                    cy={fixedSize / 2}
+                    strokeWidth="4"
```

Comment:
when the image is small, this looks really thick in comparison. could we set it conditionally based on image size?

Reply:
I've added a `stroke` prop that can take value `normal` (default) or `thin`. I don't think it should have more flexibility, because that can result in very ugly implementations. Making it conditional is quite complicated.

Reply:
2 values is ok. Maybe you can deduct them from the image size though, and not have it as a prop. Maybe more 'magic' but works more consistent. something like `const stroke = size < 50 ? thin : normal`

Reply:
Ok, I'll make it respond to `size` and set the threshold on `60`. I also added something that will make `size` always an even number (internally) because otherwise the image would not be centered properly.